package/Dockerfile

```diff
@@ -161,9 +161,9 @@ RUN curl -sLf ${!TINI_URL} > /usr/bin/tini && \
     chmod +x /usr/bin/tini /usr/bin/telemetry && \
     mkdir -p /var/lib/rancher-data/driver-metadata
 
-ENV CATTLE_UI_VERSION 2.6.9
-ENV CATTLE_DASHBOARD_UI_VERSION v2.6.9
-ENV CATTLE_CLI_VERSION v2.6.9
+ENV CATTLE_UI_VERSION 2.6.11-rc1
+ENV CATTLE_DASHBOARD_UI_VERSION 2.6.11-rc1
```

Comment:
this should be: CATTLE_DASHBOARD_UI_VERSION v2.6.11-rc1 
The UI tag does not have the "v", but the Dashboard tag does